cfpb/teachers-digital-platform
# Split the Curriculum Review Tool out into its own Django app

We want to migrate the Wagtail-dependent portions of the teachers-digital-platform app into cfgov-refresh to reduce complexity that the circular dependency between cfgov-refresh and teachers-digital-platform creates. This, the isolation of crtool into its own Django app, is a first step along that path.

## Additions

- Create a new "crtool" Django app in the teachers-digital-platform repository to hold the CRTool URLs and templates.

## Testing

1. Pull branch
1. `./setup.sh`
1. Install for local development via the `develop-apps` folder or `pip install -e`
1. Add the following to `cfgov/cfgov/urls.py`:

   ```py
        re_path(
           r'^practitioner-resources/youth-financial-education/curriculum-review/',  # noqa: E501
           include('crtool.urls')
       ),
   ```
1. Add `crtool` to `INSTALLED_APPS` in `cfgov-refresh/cfgov/cfgov/settings/base.py`
1. Test out http://localhost:8000/practitioner-resources/youth-financial-education/curriculum-review/ and http://localhost:8000/practitioner-resources/youth-financial-education/teach/activities/
1. Compare with production and ensure they look and function they same 

## Todos

In future PRs:
1. Move the Wagtail-dependent teachers_digital_platform app into cfgov-refresh/cfgov
1. Clean up the organization of the crtool app remaining in this repo

## Checklist
